Wikipedia/Code/WMFArticleContainerViewController.m

```diff
@@ -301,6 +301,13 @@ - (void)unobserveArticleUpdates {
 #pragma mark - Toolbar Setup
 
 - (void)setupToolbar {
+    
+    [self.navigationController.toolbar setBackgroundImage:[UIImage imageNamed:@"white-patch"] forToolbarPosition:UIBarPositionBottom barMetrics:UIBarMetricsDefault];
```

Comment:
please add `HAX:` comments w/ brief version of what you wrote in PR so we can get the same context in the future